src/main/java/org/cascadebot/cascadebot/events/MessageEventListener.java

```diff
@@ -0,0 +1,26 @@
+package org.cascadebot.cascadebot.events;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonPrimitive;
+import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+import org.cascadebot.cascadebot.CascadeBot;
+import org.cascadebot.cascadebot.data.Config;
+import org.cascadebot.cascadebot.utils.CryptUtils;
+
+import javax.crypto.BadPaddingException;
+import javax.crypto.IllegalBlockSizeException;
+import javax.crypto.NoSuchPaddingException;
+import javax.crypto.ShortBufferException;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+
+public class MessageEventListener extends ListenerAdapter {
+
+    @Override
+    public void onGuildMessageReceived(GuildMessageReceivedEvent event) {
+        CascadeBot.INS.getMessageReceivedRunnable().getQueue().add(event);
```

Comment:
We might want to add self-user/bot checks to this

Reply:
As talked about before, we probably do want to log bot/our user messages.

If we don't someone could make a tag to ghost ping people, and if we don't log that they'd have no way of finding out who (assuming they don't log commands, or the other events that can log it).

Reply:
:+1: